.github/workflows/verification.yml

```diff
@@ -7,8 +7,13 @@ on:
       - master
 
 jobs:
-  tests-lint:
-    name: Run tests and lint
+  lint:
+    name: Lint
+    uses: flyteorg/flytetools/.github/workflows/lint.yml@master
+    with:
+      go-version: 1.16
+  tests:
+    name: Unit Tests
```

Comment:
You can use re re usable workflow like this 
```
  tests:
    name: Unit Tests
    uses: flyteorg/flytetools/.github/workflows/tests.yml@master
    secrets:
      FLYTE_BOT_PAT: ${{ secrets.FLYTE_BOT_PAT }}
    with:
      go-version: 1.16
```

Reply:
added